data/human/human missions.txt

```diff
@@ -5547,3 +5547,228 @@ mission "Quicksilver Mixup 1"
+				goto taxi
+
+			label waiting
+			action 
+				"Blind Man from Martini: taxi questions count" ++
+			`	Deciding not to ask Douglas anything, you look out the windows of the taxi. You're in the midst of New Sydney, and skyscrapers stare down at your tiny taxi from each side of the avenue, but despite this it is still quite green and cheery, with trees on each side and people walking to work in the morning.`
+				to display
+					"Blind Man from Martini: taxi questions count" == 1
+				goto taxi
+
+			`	As the taxi drives on a highway on the edge of a small hill, the window gives you a panorama of the suburbs of New Sydney and the two skylines on each side, one marking the commercial district where the spaceport is prominent in its gleaming steel-glass architecture, and the other being the University district, a series of clean and white, flowing buildings and skyscrapers. Far away on the horizon, you can barely notice the smokestacks of factories.`
+				to display
+					"Blind Man from Martini: taxi questions count" == 2
+				goto taxi
+
+			`	The taxi enters the University district proper now, and the window shows thongs of energetic university students from the University of New Sydney travelling to and from lectures and classrooms; going over steel walkways that run from building to building above you, and on footpaths beside the streets; clutching their backpacks and data pads. Your datapad shows that University isn't actually the only university district in New Sydney. The University of Chiron's campus is on the other side of town, presumably to keep the fierce competition between the two institutions from boiling over each other.`
+				to display
+					"Blind Man from Martini: taxi questions count" == 3
+				goto taxi
+
+			label hospital
+			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of the University district. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is stark from the outside, to match the architecture of the surroundings, yet strangely colorful on the inside, as if the person building it decided that hospital interiors were far too unimaginative.`
+			`	After a short queue, you reach the reception with Douglas. The receptionist, placed behind a sterile-looking screen that looks far out of place with the colorful hallways, asks with a well-practiced smile, "How may I help you?"`
+			choice
+				`	"We're looking for Anita Pears. Where can we find her?"`
+			`	For a moment, the receptionist shuffles information arounds on her terminal. "You are her father, Douglas Pears?"`
+			`	"That's me," Douglas interjects before you can speak.`
```

Comment:
```suggestion
			`	For a moment, the receptionist shuffles information around on her terminal. "Are one of you a parent?"`
			`	"I'm the father, Douglas Pears," he replies.`
```
If the receptionist is addressing `<first>` here, asking if they're the father assumes a gender.